cosimulation/icarus/test/Makefile

```diff
@@ -0,0 +1,2 @@
+all:
+	python test_all.py
```

Comment:
I just added a Makefile to the already existing cosimulation/icarus/test directory.
This allows me to continue following the make -C pattern without worrying about the working directory.